.coveragerc

```
@@ -7,3 +7,6 @@ disable_warnings =
 
 [report]
 show_missing = True
+exclude_lines =
```

Modern coverage allows extending the built-in list of exclusions now, instead of replacing it:
```suggestion
exclude_also =
```

---

> Modern coverage allows extending the built-in list of exclusions now, instead of replacing it:

@webknjaz Thanks for suggesting it. I was about to use it myself later, as I learnt about it a while ago—https://github.com/python-poetry/poetry/pull/8736 :)